charts/newrelic-infra-operator/values.yaml

```
@@ -0,0 +1,96 @@
+# IMPORTANT: The Kubernetes cluster name
+# https://docs.newrelic.com/docs/kubernetes-monitoring-integration
+cluster: ""
+
+image:
+  repository: newrelic/newrelic-infra-operator
+  tag: ""  # Defaults to AppVersion
+  pullPolicy: IfNotPresent
+  ## It is possible to specify docker registry credentials
+  ## See https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  # pullSecrets:
+    # - name: regsecret
+
+admissionWebhooksPatchJob:
+  image:
+    repository: jettech/kube-webhook-certgen
+    tag: v1.5.0
+    pullPolicy: IfNotPresent
+  # Volume mounts to add to the job, you might want to mount tmp if Pod Security Policies
+  # Enforce a read-only root.
+  volumeMounts: []
+  #  - name: tmp
+  #    mountPath: /tmp
+
+  # Volumes to add to the job container
+  volumes: []
+  #  - name: tmp
+  #    emptyDir: {}
+
+replicas: 1
+
+resources:
+  limits:
+    memory: 80M
+  requests:
+    cpu: 100m
+    memory: 30M
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created for the job and
+  # the deployment.
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+# Configure podSecurityContext
+podSecurityContext:
+  enabled: false
+  fsGroup: 1001
+  runAsUser: 1001
+  runAsGroup: 1001
+
+# Use cert manager for webhook certs, rather than the built-in patch job
+certManager:
+  enabled: false
+
+# Use custom tls certificates for the webhook, or let the chart handle it
+# TODO: Add support for defining custom certs for the webhook here
+# customTLSCertificate: {}
+
+# If you wish to provide additional annotations to apply to the pod(s), specify
+# them here
+# podAnnotations:
+
+# Pod scheduling priority
+# Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+# priorityClassName: high-priority
+
+# Webhook timeout
+#
+# Configure how long the API server should wait for a webhook to respond before treating the call as a failure
+#
+# Ref: https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#timeouts
+timeoutSeconds: 30
+
+# Operator configuration
+# The following are the default values for the operator
+config:
+  # infraSidecarInjection: {}  # Sidecar injection can be disabled by setting this to an empty object
```

I'd rather set it to `nil`, so it is easy to disable the injection. Or actually, do we want to support disabling it? Shouldn't we disable whole operator deployment? Also` enabled: true/false` would be more standard than `infraSidecarInjection: null`.
```suggestion
  # infraSidecarInjection: null  # Sidecar injection can be disabled by setting this to an empty object
```

---

I'm fine with the `enabled`, but since the config is passed "as is", it would be operator after unmarshalling the yaml the one deciding to not run.

---

I'd drop it for now, until we actually have some other functionality in the operator.